stdlib/map.mli

```diff
@@ -129,10 +129,10 @@ module type S =
         arguments, the function [f] is used to combine them.
         This is a special case of [merge]: [union f m1 m2] is equivalent
         to [merge f' m1 m2], where
-        - [f' None None = None]
-        - [f' (Some v) None = Some v]
-        - [f' None (Some v) = Some v]
-        - [f' (Some v1) (Some v2) = f v1 v2]
+        - [f' k None None = None]
+        - [f' k (Some v) None = Some v]
+        - [f' k None (Some v) = Some v]
+        - [f' k (Some v1) (Some v2) = f k v1 v2]
```

Comment:
I think it would make sense to be even more explicit by using `_key` rather than `k` in the first three cases, and `key` in the last case.